fvtest/tril/test/IlGenTest.cpp

```diff
@@ -40,6 +40,9 @@
 
 class IlGenTest : public Tril::Test::JitTest {};
 
+// This test is currently broken on AIX, since it is not valid to use the return value of
+// compileMethodFromDetails as a function pointer.
+#if !defined(AIXPPC)
```

Comment:
Does this mean this test won't work on z/OS either? Sounds like we need the `FunctionDescriptor` trick here as well.

Reply:
While I could put the `FunctionDescriptor` trick in here, there isn't really a good place to do it with how this test is currently laid out. Since the test is calling `compileMethodFromDetails` directly, I would need to put the code for handling function descriptors in the test itself (or come up with a more proper solution than what we have right now) and I'm a bit hesitant to keep copy/pasting that hack around the codebase too much.

You're right, though, this test wouldn't work on z/OS either. Since this is a test of Tril itself, I'm not sure how important we consider this. I can add an exclude for z/OS as well if you'd like?

Reply:
We can leave it for now. There will be some more work to get stuff running on z/OS at which point we'll need to verify this regardless.